tools/elf2hunk.c

```diff
@@ -276,10 +306,20 @@ static void set_error(int err)
 #define D(x)
 #define DB2(x)
 #endif
+
+#if defined(_MSC_VER)
+#define bug(fmt,...)    fprintf(stderr, fmt , __VA_ARGS__ )
+#else
 #define bug(fmt,args...)	fprintf(stderr, fmt ,##args )
+#endif
 
 static int must_swap = -1;
 
+// hack for SHN_COMMON
+static struct hunkheader *theBSS;
```

Comment:
Original code doesn't use global variables for storing information about ELF. Is there a way to avoid defining `theBSS` and `bssIndex` as globals?